diff --git a/node_modules/react-native-video/src/Video.tsx b/node_modules/react-native-video/src/Video.tsx
index 82b73ab..d113abb 100644
--- a/node_modules/react-native-video/src/Video.tsx
+++ b/node_modules/react-native-video/src/Video.tsx
@@ -219,7 +219,7 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
         if (!uri && _source.ad?.type !== 'ssai') {
           console.log('Trying to load empty source');
         }
-        const isNetwork = !!(uri && uri.match(/^(rtp|rtsp|http|https):/));
+        const isNetwork = !!(uri && uri.match(/^(rtp|rtsp|http|https|jellyfin-multi):/));
         const isAsset = !!(
           uri &&
           uri.match(
